src/utils/common.js

```
@@ -84,6 +85,13 @@ export const popupCenter = async ({url, title, callback}) => {
       `,
     );
 
+    if (newWindow) {
+      try { 
```

ESLint

[error] Delete `·`

(prettier/prettier)